packages/h5p-examples/config.json

```diff
@@ -15,5 +15,16 @@
         "H5P.CoursePresentation": ["H5P.MathDisplay"],
         "H5P.InteractiveVideo": ["H5P.MathDisplay"],
         "H5P.DragQuestion": ["H5P.MathDisplay"]
+    },
+    "theme": {
```

Comment:
Further customization options could be:

- corner rounding other button decorations
- font sizes and font faces
- margins and paddings

Reply:
I already planed to add those.